### ISSUE TITLE
Support for tags on Kinesis Firehose delivery streams

### ISSUE THREAD

**Issue by kstark** [terraform-providers/terraform-provider-aws#4781]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

AWS seems to support [tags for Kinesis Firehose Delivery Streams](https://docs.aws.amazon.com/cli/latest/reference/firehose/tag-delivery-stream.html); it'd be helpful if that functionality was exposed in Terraform.

### New or Affected Resource(s)

* aws_kinesis_firehose_delivery_stream

### Potential Terraform Configuration

```hcl
resource "aws_kinesis_firehose_delivery_stream" "example_stream" {
    name = "example_stream"

    tags {
        ManagedBy = "terraform"
    }
}
```

**Comment by bflad** [comment]
Support for `aws_kinesis_fireshose_delivery_stream` tags has been merged and will release with version 1.47.0 of the AWS provider, sometime this week. 👍

**Comment by bflad** [comment]
This was released (4 days ago) in [version 1.47.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1470-november-26-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
